src/app/not-found.tsx

```diff
@@ -1,29 +0,0 @@
-import Link from "next/link";
```

Comment:
please don't remove the not found page, this is required instead showing the default 404